ibis-server/tests/routers/ibis/test_bigquery.py

```diff
@@ -0,0 +1,115 @@
+import pytest
+from fastapi.testclient import TestClient
+
+from app.main import app
+
+client = TestClient(app)
+
+
+@pytest.mark.bigquery
+class TestBigquery:
+
+    @pytest.fixture()
+    def manifest_str(self) -> str:
+        import base64
+        import orjson
+
+        manifest = {
+            "catalog": "my_catalog",
+            "schema": "my_schema",
+            "models": [
+                {
+                    "name": "Orders",
+                    "properties": {},
+                    "refSql": "select * from tpch_tiny.orders",
+                    "columns": [
+                        {"name": "orderkey", "expression": "o_orderkey", "type": "integer"},
+                        {"name": "custkey", "type": "integer", "expression": "o_custkey"}
+                    ],
+                    "primaryKey": "orderkey"
+                },
+                {
+                    "name": "Customer",
+                    "refSql": "select * from tpch_tiny.customer",
+                    "columns": [
+                        {"name": "custkey", "expression": "c_custkey", "type": "integer"},
+                        {"name": "name", "expression": "c_name", "type": "varchar"}
+                    ],
+                    "primaryKey": "custkey"
+                }
+            ]
+        }
+        return base64.b64encode(orjson.dumps(manifest)).decode('utf-8')
+
+    @staticmethod
+    def get_connection_info():
+        import os
+        return {
+            "project_id": os.getenv("TEST_BIG_QUERY_PROJECT_ID"),
+            "dataset_id": "tpch_sf1",
```

Comment:
```suggestion
            "dataset_id": "tpch_tiny",
```
I think it should be `tiny`, right?